triage/update_summaries.sh

```diff
@@ -55,9 +55,11 @@ bq query --allow_large_results --headless -n0 --replace --destination_table k8s-
   where
     test.failed
     and timestamp_to_sec(started) > TIMESTAMP_TO_SEC(DATE_ADD(CURRENT_DATE(), -14, 'DAY'))"
-bq extract --compression GZIP --destination_format NEWLINE_DELIMITED_JSON 'k8s-gubernator:temp.triage' gs://k8s-gubernator/triage_tests.json.gz
-gsutil cp gs://k8s-gubernator/triage_tests.json.gz triage_tests.json.gz
-gzip -df triage_tests.json.gz
+gsutil rm gs://k8s-gubernator/triage_tests/shard_*.json.gz || true
+bq extract --compression GZIP --destination_format NEWLINE_DELIMITED_JSON 'k8s-gubernator:temp.triage' gs://k8s-gubernator/triage_tests/shard_*.json.gz
+mkdir -p triage_tests
```

Comment:
Does the `triage_tests` directory inside the container need to be cleaned up at the end of the script (e.g. `rm -f triage_tests`)? 

A scenario where I think this would confound results is if the # shards *decreases* between runs. `gzip -drf triage_tests` will *overwrite* shards on subsequent runs but only if they exist.

Probably an edge-case but it should be avoided nonetheless.

Reply:
The container file system won't persist between runs. The only volume we have mounted is a secret volume mounted at `/creds` (which I don't think would write back to the secret in etcd anyways): https://github.com/kubernetes/test-infra/blob/32d7fd482e47aaf19f193332134ea86227d891f8/config/jobs/kubernetes/test-infra/test-infra-trusted.yaml#L232-L238

We do have to clean up the files in GCS though since those files persist between runs which is why I added the `gsutil rm` command at the beginning.

Reply:
👍 . I didn't realize the filesystem did not persist.

Reply:
I linked to the wrong job, but the same reasoning applies: https://github.com/kubernetes/test-infra/blob/24fcf19820b19d5903a8adda535b03c211d5f09c/config/jobs/kubernetes/test-infra/test-infra-periodics.yaml#L58-L65